src/Checkout/Mixin.js

```diff
@@ -425,21 +425,27 @@ export default {
                 this.updateBillingDetails(checkoutId)
                 this.updateShippingDetails(checkoutId)
 
-                this.activeCartCollection = Make.cloneOf(this.currentCheckout)
+                // Don't update active cart if order is complete
+                if (Tell.serverVariable(`serverStage.${checkoutId}`) < Stage.COMPLETE) {
+                    this.activeCartCollection = Make.cloneOf(this.currentCheckout)
+                }
 
                 return
             }
 
             const newCart = Make.cloneOf(this.activeCartCollection)
 
             newCart.uid = checkoutId
-            newCart.useShipping = false
+            newCart.useShipping = true
```

Comment:
This is unrelated, but should have always been `true` I believe.  If we're loading a new order, lets assume billing is the same as shipping unless the user says otherwise

Reply:
I think this was due to Skinflint, where both addresses are pre-populated from the admin system and they explicitly asked for it to default to false (IIRC)